src/CommunityToolkit/Xamarin.CommunityToolkit/Views/MediaElement/Android/MediaElementRenderer.android.cs

```diff
@@ -372,6 +375,17 @@ protected void UpdateVolume()
 
 			mediaPlayer?.SetVolume((float)MediaElement.Volume, (float)MediaElement.Volume);
 		}
+		protected void UpdateSpeed()
+		{
+			if (MediaElement == null)
+				return;
+
+			var playbackParams = new PlaybackParams();
+			playbackParams.SetSpeed((float)MediaElement.Speed);
+			if (mediaPlayer != null)
+				mediaPlayer.PlaybackParams = playbackParams;
+			var sp = mediaPlayer?.PlaybackParams.Speed;
```

Comment:
```suggestion
```
Is it needed?

Reply:
Sorry I was in a hurry and did not see that